airflow/bin/cli.py

```diff
@@ -391,8 +339,22 @@ def run(args, dag=None):
     ti = TaskInstance(task, args.execution_date)
     ti.refresh_from_db()
 
+    logger = logging.root
+    logger.handlers = []
+    if args.raw:
+        # Output to STDOUT for the parent process to read and log
+        logging.basicConfig(
+            stream=sys.stdout,
+            level=settings.LOGGING_LEVEL,
+            format=settings.LOG_FORMAT)
+    else:
+        settings.airflow_logging.pre_task_logging(ti)
+        logger = settings.airflow_logging.get_task_logger(ti)
+
+    hostname = socket.getfqdn()
+    logging.info("Running on host {}".format(hostname))
+
     if args.local:
-        print("Logging into: " + filename)
```

Comment:
If the default Airflow FileHandler has some kind of init call that is run before the other log lines we could put that there.